training/data_generator.py

```diff
@@ -27,97 +27,103 @@ class TransformationParameter(object):
     gray = 0
     transform_body_joint = True
 
-    def __init__(self):
+    #def __init__(self):
         # Nothing yet
 
-def preprocessing(train=None):
-    params = TransformationParameter()
-    params.stride = 8
-    params.crop_size_x = 368
-    params.crop_size_y = 368
-    params.target_dist = 0.6
-    params.scale_prob = 1
-    params.scale_min = 0.5
-    params.scale_max = 1.1
-    params.max_rotate_degree = 40
-    params.center_perterb_max = 40
-    params.do_clahe = False
-    params.num_parts_in_annot = 17
-    params.num_parts = 56
-    params.mirror = True
-
-    dataTransformer = DataTransformer(params)
-    # dataTransformer.initRand()
-    np = 2*(params.num_parts+1)
-    stride = params.stride
-    grid_x = params.crop_size_x / stride
-    grid_y = params.crop_size_y / stride
-    channelOffset = grid_y * grid_x
-    vec_channels = 38
-    heat_channels = 19
-    ch = vec_channels + heat_channels
-    start_label_data = (params.num_parts+1) * channelOffset
-
-    transformed_data = [] # size: params.crop_size_x * params.crop_size_y * 3
-    transformed_label = [] # size: grid_x * grid_y * np
+    def preprocessing(train=None):
```

Comment:
Hey is there a specific reason you indented all these functions? They originally were not suppose to be part of the TransformationParameter class.